d8/d8batch/README.md

```diff
@@ -0,0 +1,11 @@
+### D8 Batch
+
+#### tl;dr;
+Drupal's [Batch API](https://api.drupal.org/api/drupal/core!includes!form.inc/group/batch/8) did not change anything in Drupal 8, that is why this demo module also contains other useful examples from the Drupal 8 new features.
```

Comment:
I've moved the configuration files from the config/install to the config/optional, this way this conflict will be ignored. However I'm still not sure what would be the proper solution for storing node type/field configuration in modules.